### PR TITLE
fix result set sharing

### PR DESCRIPTION
**Pull request by colombod**

### CODE REVIEW

**Comment by colombod** [comment]
@Charles-Gagnon this should do

File: src/Microsoft.DotNet.Interactive.SqlServer/ToolsServiceKernel.cs [hunk 1]
```diff
@@ -131,7 +131,7 @@ public async Task HandleAsync(SubmitCode command, KernelInvocationContext contex
             {
                 try
                 {
-                   
+                    var results = new List<TabularDataResource>();
```
Comment (Charles-Gagnon) [inline src/Microsoft.DotNet.Interactive.SqlServer/ToolsServiceKernel.cs:134]: You're calling StoreQueryResults for each summary still which is at best unnecessary and at worst could result in multiple names with the same list (if it's even possible for the name to change...which seems unlikely but still). 

You could move that call out of the outer foreach right?

**Review by jonsequitur — APPROVED** [review]

**Review by Charles-Gagnon — APPROVED** [review]
See comment, but functionally this looks great. Thanks for the quick fix! 😄

File: src/Microsoft.DotNet.Interactive.SqlServer/ToolsServiceKernel.cs (hunk 1 above)
Comment (colombod) [reply src/Microsoft.DotNet.Interactive.SqlServer/ToolsServiceKernel.cs:134]: name collision is left to the implementation. Storing at each bach ensure nothing is lost dueto exceptions or cancellation

Comment (Charles-Gagnon) [reply src/Microsoft.DotNet.Interactive.SqlServer/ToolsServiceKernel.cs:134]: Why not call store right after creating the list then? It just seems unnecessary to call store each time when you're passing in the same list every time

**Pull request merged**